mapper/db/core.lua

```diff
@@ -139,7 +139,7 @@ function amap.db:add_follow_link(room_id, from_link, to_link)
         error("Wrong input")
     end
 
-    local curr_data = getRoomUserData(amap.curr.id, "team_follow_link")
+    local curr_data = getRoomUserData(room_to_set, "team_follow_link")
```

Comment:
Nie do końca tak to może być, bo są dwa wywołania.
Tzn. ta linia jest prawidłowa.
136 powinna natomiast w tej sytuacji wyglądać:

```lua
local room_to_set = room_id and amap.db:get_room_id(room_id) or amap.curr.id
```

Reply:
Faktycznie, jest też wywołanie z nil'em. Poprawiłem.